demo/popcha-client/web/sonar-project.properties

```diff
@@ -0,0 +1,19 @@
+sonar.organization=dedis
+sonar.projectKey=dedis_popstellar_popchawebclient
+
+sonar.python.version=3
+sonar.dynamicAnalysis=reuseReports
+sonar.core.codeCoveragePlugin=cobertura
+sonar.python.coverage.reportPaths=*coverage*.xml
+
+# Path of the test and coverage reports
+sonar.go.tests.reportPaths=./report.json
+sonar.go.coverage.reportPaths=./coverage.xml
```

Comment:
you probably don't want to refer to `Go` tests and coverage.
Check the docs for the correct params
https://docs.sonarqube.org/9.9/analyzing-source-code/test-coverage/python-test-coverage/